### PR TITLE
Added starting script for bert pruning

### PR DESCRIPTION
**Pull request by anandhu-eng**

### CODE REVIEW

File: cm-mlops/script/get-generic-python-lib/_cm.json [hunk 1]
```diff
@@ -516,6 +516,14 @@
         "CM_SCIKIT_LEARN_VERSION"
       ]
     },
+    "cupy": {
```
Comment (arjunsuresh) [inline cm-mlops/script/get-generic-python-lib/_cm.json:519]: cupy has a dependency on CUDA right?

Comment (anandhu-eng) [reply cm-mlops/script/get-generic-python-lib/_cm.json:519]: Yes its for running numpy and scipy codes in CUDA

Comment (arjunsuresh) [reply cm-mlops/script/get-generic-python-lib/_cm.json:519]: Can you please add that as a deps for cupy?

Comment (anandhu-eng) [reply cm-mlops/script/get-generic-python-lib/_cm.json:519]: Sure

**Review by arjunsuresh — APPROVED** [review]

**Pull request merged**